content/en/docs/operator.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Operators
+menu:
+  main:
+    weight: 10
+---
+
+## What is an Operator?
+
+An Operator is a method of packaging, deploying and managing a Kubernetes-native application. A Kubernetes-native application is an application that is both deployed on Kubernetes and managed using the Kubernetes APIs and kubectl tooling. An Operator is a piece of software that runs in a Pod inside the Kubernetes cluster and it interacts with the Kubernetes API. It has a reconciler loop that runs continuously, which monitors Objects (Pods, Services, ConfigMaps, or PersistentVolumes) for a desired state given by a Custom Resource Definition (CRD) and replicates that state in the cluster.
+
+Conceptually, Operators take human operational knowledge and encode it into software that is more easily shared with consumers.
+
+Operators are pieces of software that ease the operational complexity of running another piece of software. They act like an extension of the software vendor’s engineering team, watching over a Kubernetes environment (such as OpenShift Container Platform) and using its current state to make decisions in real time. Advanced Operators are designed to handle upgrades seamlessly, react to failures automatically, and not take shortcuts, like skipping a software backup process to save time.
+
+More technically, Operators are a method of packaging, deploying, and managing a Kubernetes application.
+
+A Kubernetes application is an app that is both deployed on Kubernetes and managed using the Kubernetes APIs and `kubectl` or `oc` tooling. To be able to make the most of Kubernetes, you require a set of cohesive APIs to extend in order to service and manage your apps that run on Kubernetes. Think of Operators as the runtime that manages this type of app on Kubernetes.
+
+## What is the difference between a Controller and an Operator
+
+An Operator is an application-specific controller that extends the Kubernetes API to create, configure and manage instances of complex stateful applications on behalf of a Kubernetes user. It is built on to of the basic Kubernetes resource and Kubernetes Controller along with some application-specific knowledge.
+
+We use Operators because managing stateful applications, like databases, caches and monitoring systems, is a big challenge, especially at massive scale. These systems require human operational knowledge to correctly scale, upgrade and reconfigure while at the same time protecting against data loss and unavailability.
+
+All Operators use the controller pattern, but the converse is not true. It's only an Operator if it has API extension and single-app focus in addition to the controller pattern
+
```

Comment:
Is this content from any other place and we are copying and past here?
I think we have similar content in SDK docs as well. Then, I am wondering if would not be possible we ensure that both places are with the same content.

Reply:
I have got this content from an external link. Could you please tell me in which place is it available in our docs? I will make sure the content is exactly the same

Reply:
here: https://sdk.operatorframework.io/docs/overview/

Reply:
also, see here the explanation about what is a controller https://book.kubebuilder.io/cronjob-tutorial/controller-overview.html#whats-in-a-controller.